packages/ppx_bitstring/ppx_bitstring.1.3.1/descr

```diff
@@ -0,0 +1 @@
+PPX implementation of the bitstring (https://github.com/mor1/ocaml-bitstring) extension.
```

Comment:
any chance you could shorten the first line by moving the URL to a separate paragraph?  The first line is used in the short description of `opam list` so this overflows.